Backend/app/dashboard/CommonAPIs/src/Dashboard/RiderPlatform/Ride.hs

```diff
@@ -229,3 +229,39 @@ instance HideSecrets MultipleRideSyncReq where
 validateMultipleRideSyncReq :: Validate MultipleRideSyncReq
 validateMultipleRideSyncReq MultipleRideSyncReq {..} = do
   validateField "rides" rides $ UniqueField @"rideId"
+
+---------------------------------------------------------
+-- Ticket Ride List--------------------------------------
+
+type TicketRideListAPI =
+  "kapture"
+    :> "list"
+    :> QueryParam "rideShortId" (ShortId Ride)
+    :> QueryParam "countryCode" Text
+    :> QueryParam "phoneNumber" Text
+    :> QueryParam "supportPhoneNumber" Text
+    :> Get '[JSON] TicketRideListRes
+
+newtype TicketRideListRes = TicketRideListRes
+  { rides :: [RideInfo]
+  }
+  deriving stock (Show, Generic)
+  deriving anyclass (ToJSON, FromJSON, ToSchema)
+
+data RideInfo = RideInfo
+  { rideShortId :: Maybe (ShortId Ride),
+    customerName :: Maybe Text,
+    customerPhoneNo :: Maybe Text,
+    driverName :: Text,
+    driverPhoneNo :: Maybe Text,
+    vehicleNo :: Text,
+    status :: RideStatus,
+    rideCreatedAt :: Maybe UTCTime,
+    pickupLocation :: BookingLocation,
+    dropLocation :: Maybe BookingLocation,
+    fare :: Maybe Money,
+    personId :: Id Customer,
+    classification :: Text
```

Comment:
How its related to tickets at all, can you explain?

Reply:
Classification is basically driver or customer. It was required by the Kapture team so that they can make distinction if the details are of driver or customer